vnext/Microsoft.ReactNative.Cxx/ReactPropertyBag.h

```diff
@@ -43,7 +43,7 @@
 //
 // To pass values through the ABI boundary the non-IInspectable types must be WinRT types
 // which are described here:
-// https://docs.microsoft.com/en-us/uwp/api/windows.foundation.propertytype?view=winrt-18362
+// https://docs.microsoft.com/en-us/uwp/api/windows.foundation.propertytype?view=winrt-19041
```

Comment:
Replace with version and language agnostic url:
```suggestion
// https://docs.microsoft.com/uwp/api/windows.foundation.propertytype
```